### DISCUSSION TITLE
四月更新 - 2

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#980] [通知]
## 🚀 功能

- 在播放关联视频时固定播放模式 @Richasy (#978)

## 🐛 Bug 修复

- 在发送弹幕后重置播放按钮焦点 @Richasy (#979)
- 限制播放器检测范围以避免光标无端消失 @Richasy (#975)
- 修复因播放提示导致的崩溃问题 @Richasy (#974)
- 修复关联视频的读秒问题 @Richasy (#969)
- 修复因同时弹窗导致的应用闪退问题 @Richasy (#968)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2204.2.0'>四月更新 - 2</a>.</em>

**Comment by jiyingxuan** [comment]
光标bug修复了！好快！感谢！！！
